index.html

```diff
@@ -1199,6 +1228,35 @@ <h3>Multistatus Correlation</h3>
       </p>
     </section>
 
+    <section class="informative">
+      <h3>Status Message Alterations</h3>
+
+      <p>
+When a status list uses the status messages feature, it becomes possible for
+the <a>issuer</a> to increase the type of messages that are associated with
+the <a>verifiable credentials</a> it issues over time.
+      </p>
+
+      <p>
+This feature creates a potential privacy violation where <a>holder</a> of the
+<a>verifiable credential</a> might be associated with additional status
+information that was not present when the original <a>verifiable credential</a>
+was issued. For example, initial status messages might convey "delayed" and
+"canceled", but additional status messages might be added by the <a>issuer</a>
+to convey "delayed due to non-payment" and "canceled due to illegal activity",
+which would not be apparent to the <a>holder</a> unless there was monitoring
+software operating on behalf of the holder that would warn them that the
+<a>issuer</a> intends to expose additional information about their activity.
```

Comment:
```suggestion
to convey "delayed due to non-payment" and "canceled due to illegal activity".
This change would not be apparent to the <a>holder</a> unless there was
monitoring software operating on their behalf that would warn them that
the <a>issuer</a> intends to expose additional information about their activity.
```

Reply:
Note that the holder is a not a fixed entity and the holder that presents the VC may not be the holder that requested the VC. However the issuee is a fixed entity and the above section appers to be about the issuee.